### ISSUE TITLE
difference  between training and testing

### ISSUE THREAD

**Issue by fsalmasri** [jahongir7174/YOLOv8-pt#12]
I'm using your model core, training and testing procedure. I only changed the dataloader. 

So I'm doing an overfitting of 200 images, using the same images for training and testing. 

During training my loss goes to zero. Also, when I inspect the bbx between the prediction and output the are the same. 

Only when I do testing,I see a huge difference in values or scales between the values and my validation goes to 0.
